### DISCUSSION TITLE
Can we stack range bar chart in apexchart ?

### DISCUSSION THREAD

**Discussion by SMHari** [apexcharts/apexcharts.js#2681] [Q&A]
Hi all,
Hope you are all doing great.

I just came across a usecase where I want to stack the range bar chart instead of having separate series. 
Just tried using stacked option to true but that doesn't works for the range column chart.
Posted my exact need here 
https://stackoverflow.com/questions/69375606/can-we-stack-the-range-column-chart-in-apexchart?noredirect=1#comment122624518_69375606

Wondering whether we can do the same in Apex chart as well ?

**Comment by brianlagunas** [comment]
Stacked Range Bar is not a supported chart type.

**Reply by SMHari** [reply]
Thanks Brian
IS there any hacky way to do or will that be supported in future :)

**Reply by brianlagunas** [reply]
Not that I know of, but I am very new to this project. I do know that this is not going to be implemented anytime soon. Way too many other priorities at the moment.
